posts/octavio-luna/Elasticsearch:-A-Powerful-Search-Engine-for-Backend-Engineers.md

```diff
@@ -0,0 +1,400 @@
+---
+title: 'Elasticsearch: A Powerful Search Engine for Backend Engineers'
+published: false
+description: 'Elasticsearch is a powerful search engine that can be used to search for text in documents, but it can also be used to search for structured data in JSON format. In this article, we will explore how Elasticsearch works and how it can be used to search for structured data.'
+tags: 'elasticsearch, backend, search, optimization, performance'
```

Comment:
If I remember correctly, devto only supports up to 4 tags. You might want to remove one. 
@navarroaxel do you remember if that's the case?